src/pages/waiting-room-to-car/cat-be/components/vehicle-checks-question/vehicle-checks-question.ts

```diff
@@ -16,38 +16,35 @@ export class VehicleChecksQuestionComponent implements OnChanges {
   questions: VehicleChecksQuestion[];
 
   @Input()
-  questionsToDisable: VehicleChecksQuestion[];
+  questionsToDisable: QuestionResult[];
 
   @Input()
   formGroup: FormGroup;
 
   @Input()
-  disabled: boolean;
+  index: number;
 
   @Output()
   vehicleChecksQuestionChange = new EventEmitter<QuestionResult>();
 
   @Output()
-  vehicleChecksQuestionOutcomeChange = new EventEmitter<QuestionResult>();
+  vehicleChecksQuestionOutcomeChange = new EventEmitter<QuestionOutcome>();
 
   private questionFormControl: FormControl;
   private questionOutcomeFormControl: FormControl;
 
-  static readonly questionFieldName: string = 'vehicleChecksQuestion';
-  static readonly questionOutcomeFieldName: string = 'vehicleChecksQuestionOutcome';
+  readonly questionFieldName: string = 'vehicleChecksQuestion';
+  readonly questionOutcomeFieldName: string = 'vehicleChecksQuestionOutcome';
 
   ngOnChanges(): void {
     if (!this.questionFormControl) {
       this.questionFormControl = new FormControl({ disabled: true });
-      this.formGroup.addControl(VehicleChecksQuestionComponent.questionFieldName, this.questionFormControl);
+      this.formGroup.addControl(`${this.questionFieldName}_${this.index}`, this.questionFormControl);
```

Comment:
Is there any need of a required validator for these form controls or is that logic handled on the calling page?

Reply:
Thanks Ross. The validation will be added in another PR